arussim:
  ros__parameters:
    track: "FSG.pcd"
    friction_coef: 0.5 
    state_update_rate: 100.0      # Hz

    imu:
      imu_frequency :     240.0         # Hz  Publish rate
      noise_imu_x:        0.02          # m   Noise standard deviation
      noise_imu_y:        0.02          # m   Noise standard deviation
      noise_imu_yaw:      0.01          # m   Noise standard deviation
      noise_imu_vx:       0.01          # m   Noise standard deviation
      noise_imu_vy:       0.01          # m   Noise standard deviation
      noise_imu_r:        0.005         # m   Noise standard deviation
    
    wheel_speed:
      wheel_speed_frequency :   100.0   # Hz  Publish rate
      noise_wheel_speed :       0.01    # m   Noise standard deviation

    extensometer:
      extensometer_frequency :  100.0   # Hz  Publish rate
      noise_extensometer:       0.01    # m   Noise standard deviation